### PR TITLE
Refactor ashraesizer.py

### PR DESCRIPTION
**Pull request by adr-ban**
Refactor ASHRAE sizer:
•	Transition to cfg to load constants
•	Changed names of variables to include units
•	Removed semi-colons
•	Added heat capacity and storage volume calculations

### CODE REVIEW

File: ashraesizer.py [hunk 1]
```diff
@@ -125,16 +125,53 @@ def primaryCurveAshrae(self, peakFlowTable):
...
         # simpleRecoveryTons = self.People * (peakFlowTable[:len(peakFlowTable),1]) / \
-        #         (peakFlowTable[:len(peakFlowTable),0]) * 60 * self.rhoCp * (self.storageT - self.incomingT)/12000;
+        #         (peakFlowTable[:len(peakFlowTable),0]) * 60 * rhoCp * (self.storageT_F - self.incomingT_F)/12000;
                 
-        return [primaryVol, accurateRecoveryTons];
-     
\ No newline at end of file
+        return [primaryVol, accurateRecoveryTons]
+     
+    def tonsRecoveryForMaxDaily(self, peakFlowTable, accurateRecoveryTons):
```
Comment (pmskintner) [inline ashraesizer.py:141]: accurateRecoveryTons isn't used in this function, can we remove it from the function definition?

File: ashraesizer.py [hunk 2]
```diff
@@ -125,16 +125,53 @@ def primaryCurveAshrae(self, peakFlowTable):
...
+        
+        # Find the recoveryHours below and above the defined compressor run time
+        lowerRecoveryHours = recoveryHours[recoveryHours < self.compRuntime_hr].max()
+        upperRecoveryHours = recoveryHours[recoveryHours > self.compRuntime_hr].min()
+        
+        # Find the indices surrounding the defined compressor run time
+        hoursIndices = np.where((recoveryHours == upperRecoveryHours) | (recoveryHours == lowerRecoveryHours))
+        
+        recoveryTimes = recoveryHours[hoursIndices]
+        recoveryTons = simpleTons[hoursIndices]
```
Comment (pmskintner) [inline ashraesizer.py:157]: I think all of this is unnecessary. It looks like you're trying to pin point the indices greater than and less than recovery tons and hours, but np.interp will do this for you.

File: ashraesizer.py [hunk 3]
```diff
@@ -125,16 +125,53 @@ def primaryCurveAshrae(self, peakFlowTable):
...
+        
+        lowerRecoveryTons = accurateRecoveryTons[accurateRecoveryTons <= heatCap].max()
+        upperRecoveryTons = accurateRecoveryTons[accurateRecoveryTons >= heatCap].min()
+
+        recoveryIndices = np.where((accurateRecoveryTons == upperRecoveryTons) | (accurateRecoveryTons == lowerRecoveryTons))
+
+        Volumes = primaryVol[recoveryIndices]
+        accurateRecoveries = accurateRecoveryTons[recoveryIndices]
+
+        minStorage_gal = np.interp(heatCap, Volumes, accurateRecoveries)
```
Comment (pmskintner) [inline ashraesizer.py:175]: Same comment about np.interp

File: ashraesizer.py (hunk 1 above)
Comment (adr-ban) [reply ashraesizer.py:141]: I agree. I'll remove it.

File: ashraesizer.py (hunk 2 above)
Comment (adr-ban) [reply ashraesizer.py:157]: You're right. I'll remove the unnecessary pieces.

File: ashraesizer.py (hunk 3 above)
Comment (adr-ban) [reply ashraesizer.py:175]: Agreed.

**Pull request merged**